files/default.conf

```diff
@@ -40,6 +40,12 @@ server {
     root /var/www/html;
     index index.html;
 
+    add_header Content-Security-Policy "default-src *; style-src 'self' 'unsafe-inline'";
+    add_header Strict-Transport-Security 'max-age=6307200; includeSubDomains; preload';
+    add_header X-Content-Type-Options nosniff;
+    add_header X-Frame-Options "SAMEORIGIN";
```

Comment:
`X-Frame-Options` like this will prevent people from `iframe`ing our support documentation, and I'd rather continue letting them do it. I may have to eat my words later if the burden of people complaining about issues flagged by automated security scanning tools outweighs the benefit we get from allowing the community to reuse our content inside `iframes`.